packages/underscore-redirects/src/print.ts

```diff
@@ -0,0 +1,26 @@
+import type { RedirectDefinition } from './redirects';
+
+export function print(
+	definitions: RedirectDefinition[],
+	minInputLength: number,
+	minTargetLength: number
+) {
+	let _redirects = '';
+
+	// Loop over the definitions
+	definitions.forEach((defn, i) => {
```

Comment:
nit: `for..of` is usually [faster](https://perf.link/#eyJpZCI6ImRzZDZxMjVwcTl0IiwidGl0bGUiOiJGaW5kaW5nIG51bWJlcnMgaW4gYW4gYXJyYXkgb2YgMTAwMCIsImJlZm9yZSI6ImNvbnN0IGRhdGEgPSBbLi4uQXJyYXkoMTAwMDAwKS5rZXlzKCldIiwidGVzdHMiOlt7Im5hbWUiOiJGaW5kIGl0ZW0gMTAwIiwiY29kZSI6ImxldCBzdHJpbmcgPSBcIlwiO1xuXG5kYXRhLmZvckVhY2goeCA9PiBzdHJpbmcgKz0geCkiLCJydW5zIjpbMTA4LDk5LDEwOCwxMDgsMTA4LDEwOCwxMDgsMTE3LDExNywxMDgsMTA4LDEwOCwxMDgsMTA4LDEwOCwxMDgsMTA4LDEwOCw5OSwxMDgsMTA4LDEwOCw5OSwxMDgsMTA4LDEwOCwxMDgsMTA4LDEwOCwxMDgsMTE3LDEwOCwxMDgsOTksMTA4LDEwOCwxMDgsMTA4LDEwOCwxMDgsMTA4LDEwOCwxMDgsMTA4LDEwOCwxMDgsMTA4LDExNywxMDgsMTE3LDEwOCwxMTcsMTE3LDEwOCwxMDgsMTA4LDEwOCwxMDgsMTA4LDEwOCwxMDgsMTA4LDEwOCwxMTcsMTA4LDEwOCwxMDgsMTA4LDExNywxMDgsMTA4LDEwOCwxMDgsMTE3LDExNywxMDgsMTE3LDk5LDEwOCwxMTcsMTE3LDEwOCwxMDgsMTA4LDExNywxMTcsMTE3LDExNywxMTcsMTA4LDEwOCwxMTcsMTE3LDExNywxMDgsMTA4LDEwOCwxMTcsMTE3LDEwOF0sIm9wcyI6MTA5fSx7Im5hbWUiOiJGaW5kIGl0ZW0gMjAwIiwiY29kZSI6ImxldCBzdHJpbmcgPSBcIlwiO1xuZm9yIChjb25zdCBzIG9mIGRhdGEpIHtcbiAgc3RyaW5nICs9IHM7XG59IiwicnVucyI6WzEzNSwxNjIsMTcxLDEyNiwyMTYsMTQ0LDEzNSwxODksMTgwLDEzNSwxNjIsMTgwLDE5OCwyMTYsMTQ0LDE4OSwxMDgsMjE2LDEzNSwyMTYsMTM1LDE4MCwyMDcsMjA3LDE1MywyMDcsMjE2LDE4MCwyMTYsMTUzLDE5OCwxODksMTk4LDE4OSwyMTYsMTI2LDEzNSwxMzUsMTYyLDE4MCwyMDcsMTM1LDEzNSwyMTYsMTM1LDIxNiwxOTgsMTgwLDEzNSwxODAsMTUzLDEzNSwxMzUsMjE2LDE5OCwxMzUsMTUzLDE2MiwyMTYsMTgwLDE4MCwxNDQsMTM1LDIxNiwyMjUsMTgwLDEzNSwxMzUsMTQ0LDIwNywxMzUsMTQ0LDE4OSwxODAsMjM0LDE5OCwyMTYsMjE2LDE0NCwxMzUsMTg5LDIxNiwxODAsMTcxLDE0NCwxNTMsMjE2LDIxNiwxODAsMTgwLDIwNywxODAsMjE2LDIxNiwxMjYsMTM1LDE2MiwyMTYsMTM1LDE1M10sIm9wcyI6MTc0fV0sInVwZGF0ZWQiOiIyMDIzLTA1LTI1VDA5OjA0OjQ0LjY3OVoifQ%3D%3D) than `.forEach`